src/app-common/components/common/search/search.action.js

```diff
@@ -0,0 +1,21 @@
+export function searchByTypeAction(selectedSearchType){
+    return {
+        type: 'SEARCH_MOVIES_BY_TYPE',
+        payload: selectedSearchType
+    }
+}
+
+export function getSearchValAction(searchValue){
+    return {
+        type: 'GET_SEARCH_VALUE',
+        payload: searchValue
+    }
+}
+
+export function searchMoviesAction(movies, countMovies){
+    return {
+        type: 'SEARCH_MOVIES',
+        payload: movies,
```

Comment:
We don't need to set movies here, because we don't use _movies_ from search.reducer.js in any our component.